boards/arm/mimxrt1170_evk/Kconfig.defconfig

```diff
@@ -52,4 +52,20 @@ config ETH_MCUX_PHY_RESET
 
 endif # NETWORKING
 
+if MIPI_DSI
+
+config RM68200
+	default y
```

Comment:
should not be required if display driver defaults to y based on DT compat being okay

Reply:
Without this `CONFIG_RM68200` is not set